lib/rspec/core/example_group.rb

```diff
@@ -381,9 +399,9 @@ def self.find_and_eval_shared(label, name, inclusion_location, *args, &customiza
       # @!endgroup
 
       # @private
-      def self.subclass(parent, description, args, registration_collection, &example_group_block)
+      def self.subclass(parent, description_args, metadata, registration_collection, &example_group_block)
```

Comment:
Where `description_args` is an array with either one or two elements that describe this example group, and `metadata` is already transformed to a `Hash`.